workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -262,7 +262,7 @@ protected virtual void HandleWorkerConnectionFailure()
                 ConnectionErrorReason.ExceededMaximumRetries);
         }
 
-        private static string CreateNewWorkerId(string workerType)
+        protected static string CreateNewWorkerId(string workerType)
```

Comment:
why make it protected?

Reply:
It can't be accessed from the Mobile package or the Playground when `private`.

Reply:
do we need to?

Reply:
We are overriding the `GetReceptionistConfig()` where we need to give a `workerId` in the `ReceptionistConfig` object